pipelines/DRPFakes.yaml

```diff
@@ -35,7 +35,7 @@ imports:
     - TE2
     - wPerp
 tasks:
-  processCcdWithFakes:
+  detectorVisitFakes:
```

Comment:
This new name doesn't seem to capture what this task does. Would singleFrameWithFakes work better here?